docs/vdc/networking-virtual-datacenter.md

```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 
 #### Security infrastructure
-Security infrastructure, in the context of a VDC, is mainly related to the segregation of traffic in the VDC's specific virtual network segment, and how to control ingress and egress flows throughout the VDC. Azure is based on multi-tenant architecture that prevents unauthorized and unintentional traffic between deployments, using Virtual Network (VNet) isolation, access control lists (ACLs), load balancers, and IP filters, along with traffic flow policies. Network address translation (NAT) separates internal network traffic from external traffic.
+In the context of a VDC, security infrastructure mainly relates to traffic segregation in the VDC's specific virtual network segment and how to control ingress and egress flows throughout the VDC. Azure is based on multi-tenant architecture that prevents unauthorized and unintentional traffic between deployments. It uses virtual network isolation, access control lists (ACLs), load balancers, IP filters, and traffic flow policies. Network address translation (NAT) separates internal network traffic from external traffic.
 
 The Azure fabric allocates infrastructure resources to tenant workloads and manages communications to and from virtual machines (VMs). The Azure hypervisor enforces memory and process separation between VMs and securely routes network traffic to guest OS tenants.
 
 #### Connectivity to the cloud
-The VDC needs connectivity with external networks to offer services to customers, partners and/or internal users. This usually means connectivity not only to the Internet, but also to on-premises networks and data centers.
+The VDC needs connectivity with external networks to offer services to customers, partners, and internal users. It usually needs connectivity not only to the internet but also to on-premises networks and datacenters.
 
-Customers can build their security policies to control what and how specific VDC hosted services are accessible to/from the Internet using [Azure Firewall][AzFW] or Network Virtual Appliances, and custom routing policies and network filtering ([User Defined Routing][UDR] and [Network Security Groups][NSG]). It's recommended that all Internet facing resources be additionally protected by [**Azure DDOS Protection Standard**][DDOS].
+Customers can build their security policies to control what and how specific VDC-hosted services are accessible to or from the internet with [Azure Firewall][AzFW] or network virtual appliances, custom routing policies, and network filtering. See information on [user-defined routes][UDR] and [network security groups][NSG]. We recommend that all internet-facing resources be additionally protected by [Azure DDoS Protection Standard][DDOS].
```

Comment:
This paragraph is kind of a mess. It should be something like "Customers  control which services have access to and are accessible from the public internet using [Azure Firewall][AzFW] or other types of virtual network appliances (NVAs), custom routing policies using [user-defined routes][UDR], and network filtering using [network security groups][NSG]. We recommend that all internet-facing resources also be protected by the [Azure DDoS Protection Standard][DDOS]."